src/components/SpaceProposalVotesListItem.vue

```diff
@@ -106,7 +106,7 @@ const balanceFormatted = computed(() => {
       <BaseButtonIcon
         v-if="vote.reason !== '' && props.proposal.privacy !== 'shutter'"
         v-tippy="{
-          content: urlify(vote.reason),
+          content: urlify(vote.reason.replace(/^[\s\n]+|[\s\n]+$/g, '')),
```

Comment:
Shouldn't we remove it before saving it to the HUB though?

Reply:
This will be the cleanest way to do things. In that case, we have to fo it for every text input, such as proposal body, etc…

And unless we retroactively clean all the existing fields, this pr is still needed